src/utils/__tests__/resolve_url.test.ts

```diff
@@ -27,19 +27,19 @@ describe("utils - resolveURL", () => {
     expect(resolveURL("http://toto.com/a"))
       .toBe("http://toto.com/a");
     expect(resolveURL("http://toto.com/a", "b/c/d/", "g.a"))
-      .toBe("http://toto.com/a/b/c/d/g.a");
+      .toBe("http://toto.com/b/c/d/g.a");
   });
 
   it("should ignore empty strings when concatenating multiple URLs", () => {
     expect(resolveURL("", "http://toto.com/a", ""))
       .toBe("http://toto.com/a");
     expect(resolveURL("http://toto.com/a", "b/c/d/", "", "g.a"))
-      .toBe("http://toto.com/a/b/c/d/g.a");
+      .toBe("http://toto.com/b/c/d/g.a");
   });
 
   it("should remove a leading slash if one", () => {
     expect(resolveURL("http://toto.com/a", "/b/c/d/", "/", "/g.a"))
-      .toBe("http://toto.com/a/b/c/d/g.a");
+      .toBe("http://toto.com/g.a");
```

Comment:
I agree 👍 , it should be the correct expecting result if I red correctly the spec.